sdk/core/ci.yml

```diff
@@ -46,6 +46,7 @@ extends:
       safeName: AzureCore
       triggeringPaths:
       - /sdk/resourcemanager/
+      - /eng/
```

Comment:
Not sure that we want this to hit EVERYTHING in `/eng`.    We should definitely include the buid props, build targets, and package versions.   I don't see a reason why we would run it for every update of the common tools sync for scripts, etc.

Thoughts?

Reply:
I don't have any other reason other than I followed up and was like

> I'll add eng/!

And didn't think deeper than that. 

> We should definitely include the buid props, build targets, and package versions

I'll narrow this to the props and targets files under eng/`. When you say `package versions` where do you mean in the eng/ folder?

Reply:
It was `Packages.Data.props` that was in my head.   That's where we keep the central version for all dependencies.